app/models/pageflow/account.rb

```diff
@@ -3,7 +3,7 @@ class Account < ActiveRecord::Base
     include FeatureTarget
 
     has_many :users
```

Comment:
I think it's the same situation for users, right? Maybe we can handle that in one go?